indexer/search_string_utils.cpp

```diff
@@ -283,8 +320,42 @@ class StreetsSynonymsHolder
   bool MatchPrefix(UniString const & s) const { return m_strings.HasPrefix(s); }
   bool FullMatch(UniString const & s) const { return m_strings.HasKey(s); }
 
+  template <typename DFA>
+  bool MatchWithMisprints(DFA const & dfa) const
+  {
+    using TrieIt = Trie::Iterator;
+    using State = pair<TrieIt, typename DFA::Iterator>;
+
+    auto const trieRoot = m_strings.GetRootIterator();
+
+    queue<State> q;
+    q.emplace(trieRoot, dfa.Begin());
+
+    while (!q.empty())
+    {
+      auto const p = q.front();
+      q.pop();
+
+      auto const & trieIt = p.first;
+      auto const & dfaIt = p.second;
+
+      if (dfaIt.Accepts())
+        return true;
+
+      trieIt.ForEachMove([&q, &dfaIt](UniChar const & c, TrieIt const & curDfaIt) {
```

Comment:
s/trieIt/curTrieIt/
s/curDfaIt/nextTrieIt/

Reply:
and rename other iterator-related variable too, please

Reply:
done